src/api/token.py

```diff
@@ -0,0 +1,57 @@
+""" Token Related functionalities"""
+
+import requests
+from src.constants.constants import SWAN_API, TOKEN_VALIDATION
+from src.exceptions.exceptions import HTTPError, RequestError
+
+
+def validate_token(token):
```

Comment:
It's better we put this validate part into SDK init prat, the user puts a token into the .env file, and we do validate + get JWT Token when init the sdk